handler.py

```diff
@@ -83,15 +92,20 @@ def report_cost(event, context):
     # Sort the map by yesterday's cost
     most_expensive_yesterday = sorted(cost_per_day_by_service.items(), key=lambda i: i[1][-1], reverse=True)
 
+    service_names = [k for k,_ in most_expensive_yesterday[:5]]
+    longest_name_len = len(max(service_names, key = len))
```

Comment:
I think this can be written shorter like so?

```suggestion
    longest_name_len = len(max(service_names.keys()[:5], key=len))
```

Reply:
Pretty sure `most_expensive_yesterday` is a list of tuples (from line 90 using `.items()`) , otherwise `.keys()` would work. I ran into that and was miffed initially. I should have said something like `[name for name, value in ...]` in my line 92.

Reply:
Ah yep, thanks for checking.